manticore/__main__.py

```diff
@@ -1,75 +1,85 @@
 import sys
-import os
 import argparse
 import logging
-import time
-import types
-import cPickle
-from multiprocessing import Manager, Pool, Process
-from threading import Timer
-from core.smtlib import Expression
-from manticore import Manticore
-try:
-    import psutil
-except ImportError:
-    pass
+
+from .manticore import Manticore
```

Comment:
we actually don't want to use a relative import here. the manticore cli is ideally just a regular Manticore client